src/ostree/ot-admin-builtin-deploy.c

```diff
@@ -90,6 +92,18 @@ ot_admin_builtin_deploy (int argc, char **argv, OstreeCommandInvocation *invocat
     return FALSE;
   }
 
+  if (opt_kernel_arg_none && opt_kernel_argv_delete)
+  {
```

Comment:
GNU style has two spaces for indent here.  But I think we'll eventually do https://github.com/ostreedev/ostree/pull/2565 anyways; feel free to fix in a later patch though!